metricflow/test/integration/test_configured_cases.py

```diff
@@ -134,6 +134,34 @@ def double_data_type_name(self) -> str:
         """Return the name of the double data type for the relevant SQL engine"""
         return self._sql_client.sql_engine_attributes.double_data_type_name
 
+    def render_dimension_template(self, dimension_name: str, entity_path: Sequence[str] = ()) -> str:
+        """Renders a template that can be used to retrieve a dimension.
+
+        For example:
+
+         "{{ render_dimension_template('country_latest', entity_path=['listing']) }}"
+
+         ->
+
+         "{{ dimension('country_latest', entity_path=['listing'] }}"
+
+         This is needed as the where_constraint field in the definition files are rendered twice through Jinja - once
+         by the test framework, and again by MF.
+        """
+        return f"{{{{ dimension('{dimension_name}', entity_path={repr(entity_path)}) }}}}"
+
+    def render_entity_template(self, entity_name: str, entity_path: Sequence[str] = ()) -> str:
+        """Similar to render_dimension_template() but for entities."""
+        return f"{{{{ entity('{entity_name}', entity_path={repr(entity_path)}) }}}}"
+
+    def render_time_dimension_template(
+        self, time_dimension_name: str, time_granularity: str, entity_path: Sequence[str] = ()
+    ) -> str:
+        """Similar to render_dimension_template() but for time dimensions."""
+        return (
+            f"{{{{ time_dimension('{time_dimension_name}', '{time_granularity}', entity_path={repr(entity_path)}) }}}}"
```

Comment:
Yikes. Nothing for it..... but that's a mark in favor of not using jinja if we don't end up needing to do so.

Reply:
Up for other ideas!